apps/wallet/src/shared/utils/index.ts

```diff
@@ -26,6 +27,9 @@ export function usePageView() {
             url: location.pathname,
         });
 
+        growthbook.setAttributes({
```

Comment:
Keep in mind that old wallet versions exist where this isn't being set, so you'll want to make sure that your conditions account for that.